### PR TITLE
Ananya lib

### PR DESCRIPTION
**Pull request by ananya-anni**

### CODE REVIEW

**Review by Shubhani25 — APPROVED** [review]

**Review by anushka1403 — APPROVED** [review]
reviewed

File: LibrarySystemService/src/main/java/com/target/ready/library/system/service/LibrarySystemService/Controller/CategoryController.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package com.target.ready.library.system.service.LibrarySystemService.Controller;
+
+import com.target.ready.library.system.service.LibrarySystemService.Entity.Category;
+import com.target.ready.library.system.service.LibrarySystemService.Service.CategoryService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("library_service_api/v2")
```
Comment (akarsh7791) [inline LibrarySystemService/src/main/java/com/target/ready/library/system/service/LibrarySystemService/Controller/CategoryController.java:9]: "library/v2" how about this? Everyone know its an api and its a service.

File: LibrarySystemService/src/main/java/com/target/ready/library/system/service/LibrarySystemService/Controller/LibrarySystemController.java [hunk 2]
```diff
@@ -36,5 +36,8 @@ public List<Book> getAllBooks(@RequestParam(value = "pageNumber", defaultValue =
         }
     }
 
+    @PostMapping("addBook")
```
Comment (akarsh7791) [inline LibrarySystemService/src/main/java/com/target/ready/library/system/service/LibrarySystemService/Controller/LibrarySystemController.java:39]: No verb and camel case for api name.

File: LibrarySystemService/src/main/java/com/target/ready/library/system/service/LibrarySystemService/Controller/CategoryController.java (hunk 1 above)
Comment (ananya-anni) [reply LibrarySystemService/src/main/java/com/target/ready/library/system/service/LibrarySystemService/Controller/CategoryController.java:9]: Sure

File: LibrarySystemService/src/main/java/com/target/ready/library/system/service/LibrarySystemService/Controller/LibrarySystemController.java (hunk 2 above)
Comment (ananya-anni) [reply LibrarySystemService/src/main/java/com/target/ready/library/system/service/LibrarySystemService/Controller/LibrarySystemController.java:39]: Done

**Pull request merged**